test/test_pte.cpp

```diff
@@ -123,34 +128,40 @@ int main(int argc, char *argv[]) {
     RView sie_v("sie", NPTS);
     RView temp_v("temp", NPTS);
     RView press_v("press", NPTS);
+    Rview scratch_v("scratch", NTRIAL * nscratch_vars);
```

Comment:
should be `RView`, the p9 test caught this. I can fix and push, just not sure how you feel about others pushing to your branch w/o you knowing.

Reply:
sure go for it.  looking at the thing @Yurlungur pointed out.  thanks.